src/player/player.controller.js

```diff
@@ -25,15 +26,31 @@ async function getPlayerById(playerId) {
 }
 
 function getDepositQuantity(req) {
-  const { quantity } = req.body;
+  const quantity = Number(req.query.quantity);
   if (!quantity) throw new ValidationError('Missing field: quantity');
-  if (!Number.isInteger(quantity)) throw new ValidationError('Quantity isn\'t a integer');
+  if (Number.isNaN(quantity)) throw new ValidationError('Quantity isn\'t a integer');
   if (quantity < 0) throw new ValidationError('Quantity must be a positive integer');
   return quantity;
 }
 
-function addFundsTo(player, quantity) {
-  return database.player.incrementBalanceOfPlayer(player.uuid, quantity);
+async function addFundsTo(player, quantity) {
+  await database.player.incrementBalanceOfPlayer(player.uuid, quantity);
+  return `Added ${quantity} to balance succesfully`;
+}
+
+async function editPlayer(player, name, password) {
+  console.log(player);
```

Comment:
Those console.log are for debug purpose? They should be here?

Reply:
sorry, fixed